layouts/partials/install.html

```diff
@@ -2,114 +2,108 @@
   <div class="container">
 
     <h2 class="content-center">Install IPFS</h2>
-    <h5 class="content-center" style="margin-bottom: 60px;">No matter how you want to use IPFS, there's an install
-      option for you. Unsure which to choose? Start with <a href="https://github.com/ipfs-shipyard/ipfs-desktop"
-        target="_blank"> IPFS Desktop</a>.</h5>
+    <h5 class="content-center" style="margin-bottom: 60px;">Join the future of the web right now &mdash; just choose the option that's right for you.</h5>
 
+    <h3 class="content-center">Store and share files</h3>
     <div class="grid-flex-container">
-      <div class="grid-flex-cell-1of2">
-        <a href="https://github.com/ipfs-shipyard/ipfs-desktop" target="_blank">
+
+      <div class="grid-flex-cell-1of1">
           <div class="install-item">
             <div class="install-item-image">
-              <img src="/images/ipfs-desktop-hex.png" />
+              <a href="https://github.com/ipfs-shipyard/ipfs-desktop" target="_blank" rel="noopener noreferrer"><img src="/images/ipfs-desktop-hex.png" /></a>
             </div>
             <div class="install-description">
-              <h4>Desktop app</h4>
+              <a href="https://github.com/ipfs-shipyard/ipfs-desktop" target="_blank" rel="noopener noreferrer"><h4>IPFS Desktop</h4></a>
+              <h5>IPFS for everyone</h5>
```

Comment:
Why noreferrer?

Reply:
Because I completely auto-piloted on "need to open this in a new tab". Thanks for catching that.